lib/rules/destructuring-assignment.js

```diff
@@ -38,21 +76,22 @@ module.exports = {
   create: Components.detect((context, components, utils) => {
     const configuration = context.options[0] || DEFAULT_OPTION;
     const ignoreClassFields = (context.options[1] && (context.options[1].ignoreClassFields === true)) || false;
+    const sfcParams = createSFCParams();
```

Comment:
hm, does eslint itself not have utilities for determining the relationship to function params?